fastplotlib/graphics/selectors/_linear.py

```diff
@@ -340,7 +350,7 @@ def _get_selected_index(self, graphic):
 
         if "Heatmap" in graphic.__class__.__name__ or "Image" in graphic.__class__.__name__:
             # indices map directly to grid geometry for image data buffer
-            index = self.selection() - offset
+            index = self._selection() - offset
```

Comment:
once `_selection` is defined in the construction just access it using the property.